infraohjelmointi_api/services/CoordinatorNoteService.py

```diff
@@ -0,0 +1,22 @@
+from django.shortcuts import get_object_or_404
+from ..models import CoordinatorNote
+class CoordinatorNoteService:
+
+    def create(self, request):
```

Comment:
Function can be refactored to only take in the fields required to create a CoordinatorNoteInstance. Makes it easier to use this service in other parts of the code where the request context is not available. Can check other services for example.

Reply:
Decided to delete the coordinatorNoteService because it seems to work without it also